pkg/obfuscator/ip.go

```diff
@@ -63,7 +63,7 @@ func (o *ipObfuscator) replace(s string) string {
 				replacement := gen.generateReplacement(cleaned, o.ReplacementTracker)
 				output = strings.ReplaceAll(output, m, replacement)
 				// also add the original (non-cleaned) string, this is only used for human review in the final report
-				o.ReplacementTracker.AddReplacement(m, replacement)
+				o.ReplacementTracker.AddReplacement(cleaned, replacement)
```

Comment:
Since we now care about the uniquenes of keys, 10.131.0.24 != 10-131-0-24

Reply:
it is that way, so we can later verify in the e2e test that this was replaced. When you have the cleaned version only, then it becomes difficult to reconstruct how it was looking before.

For a human it's also more useful to understand what the actual input was, not a cleaned version. Why do you think we're interested in the uniqueness of the key?

Reply:
Those are good point, original input will be retained in the report.